com.unity.cinemachine/Samples~/Shared Assets/Scripts/SimplePlayerAnimator.cs

```diff
@@ -38,15 +49,17 @@ void Start()
             if (m_Controller != null)
             {
                 // Install our callbacks to handle jump and animation based on velocity
-                m_Controller.StartJump += () => m_Animator.SetTrigger("Jump");
-                m_Controller.EndJump += () => m_Animator.SetTrigger("Land");
+                m_Controller.StartJump += () => OnJump(true);
+                m_Controller.EndJump += () => OnJump(false);
                 m_Controller.PostUpdate += (vel, jumpAnimationScale) => UpdateAnimation(vel, jumpAnimationScale);
             }
         }
         
-        // We use LateUpdate so we normally don't have to worry about script execution order:
-        // we can assume that the player has already been moved.
-        void LateUpdate()
+        /// <summary>
+        /// We use LateUpdate so we normally don't have to worry about script execution order:
+        /// we can assume that the player has already been moved.
```

Comment:
First person shouldn't be used in descriptions that fall into the API docs. A bit tricky for me here to rephrase while keeping your intention. If you can easily rephrase, that would be nice, thanks :)